add baselines json

MaddyDev opened the pull request:
Will add instructions to the on-call wiki for the on-call person to keep tabs on the perf run times.

Charles-Gagnon commented on performance/PerformanceBaselines.json:
```diff
@@ -0,0 +1,79 @@
+    "PerformanceRuntime": {
+        "InputBinding": {
+            "TestName": "GetProductsTest",
+            "RunTimeAverage": 2.081
```
Please add units to these property names (e.g. `RunTimeAverageSec`)

MaddyDev commented:
Added the [monitor perf pipeline](https://msdata.visualstudio.com/SQLToolsAndLibraries/_wiki/wikis/SQLToolsAndLibraries.wiki/52049/On-Call-Duties?anchor=monitor-performance-pipeline) section in on-call wiki, I've added 100 sec ~> 1.5 min as the threshold there for us to investigate the deviation, we can discuss what's a good number and update it there in our on-call handoff sync.
